app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Add a New Review for <%=@train.name%></h1>
+
+<%= form_for [@train, @review] do |f| %>
+  <%= f.label :rating, "Rating" %>
+  <%=  f.collection_radio_buttons(
+      :rating, [[1, '1'] ,[2, '2'],[3, '3'],[4, '4'],[5, '5']], :first, :last
+    ) do |b|
+      b.label { b.radio_button + b.text }
+      end %>
```

Comment:
even though right about we use |f| as the variable for form using |b| for button here is a bit off convention. I'll leave it up to you whether to change it or not, but it's the kind of habit I would strongly advise against getting into.